ci/github-actions/cipher.rb

```diff
@@ -0,0 +1,54 @@
+#!/usr/bin/env ruby
```

Comment:
looks like this is a new file to provide some basic salting/hashing of our passwords

Reply:
We've had some issues with openssl in the past so @anyaelena added this as a sanity check during the gem builds.